test/show.jl

```diff
@@ -880,3 +880,28 @@ end
     @test replstr(zeros(Complex{Int}, 1, 2, 1)) ==
         "1×2×1 Array{Complex{$Int},3}:\n[:, :, 1] =\n 0+0im  0+0im"
 end
+
+@testset "Array printing with limited rows" begin
+    buf = IOBuffer()
+    arrstr(A, rows) = (Base.showarray(IOContext(buf, displaysize=(rows, 80), limit=true),
+                                      A, false, header=true);
+                       String(take!(buf)))
```

Comment:
Perhaps a long-form definition would be better? The indentation is somewhat confusing.